lib/stimulus_reflex/logger.rb

```diff
@@ -12,6 +12,7 @@ def initialize(reflex)
     def print
       return unless config_logging.lambda?
       puts
+      set_config_lambda_binding
```

Comment:
Hm, not quite what I meant. This will still do the `.eval` on every reflex. The memoization only matters for multiple operations in the reflex.

Reply:
Ah, wait. Does it mean the purpose of memoization was to do `eval` once?

Reply:
@RolandStuder My apologies. I misunderstood the intentions there. I thought what was memoized was `StimulusReflex.config.logging` but there was no `||=` which I missed. The trick was to avoid `eval` there. Restored the previous version!

Surely, we can improve on the `eval` and move it to `Configuration` but as I mentioned [here](https://github.com/hopsoft/stimulus_reflex/pull/337#discussion_r523794571) definig a setter did not really work out.

I am all good to merge this PR in a current shape ✅ Any improvements can be done afterwards. What do you think?